### PR TITLE
Address Comments + Fix Linting

### PR DESCRIPTION
**Pull request by quinndyer**

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -1,58 +1,42 @@
 const argv = require('minimist')(process.argv.slice(2));
```
Comment (quinndyer) [inline src/index.ts]: I could not figure out what I had to import from minimist to replace this require with an import to make eslint happy...

docs: https://github.com/substack/minimist

(this is the single lint error left)

**Comment by adatta02** [comment]
@quinndyer looks like its also complaining about:
```
46:30  warning  Unexpected any. Specify a different type        @typescript-eslint/no-explicit-any
```
https://github.com/quinndyer/demo/blob/fix-linting/src/index.ts#L46

In phpstorm if you hold CRTL and then click the "on" method it should jump you to the types, looks like data is a Buffer
```
on(event: "data", listener: (data: Buffer) => void): this;
```

File: src/index.ts (hunk 1 above)
Comment (adatta02) [reply src/index.ts]: Going off https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/no-var-requires.md

It looks like it wants:
```
import minimist = require("minimist");
const argv = minimist(process.argv.slice(2));
```

Comment (quinndyer) [reply src/index.ts]: ah makes sense

**Comment by quinndyer** [comment]
Added TLS which is working to actually get a 200 response from setfive website instead of 300 moved one. But for some reason it says error part way through the HTML response? Unsure why. (This is a GET request to https://setfive.com)

image

Finishes to completion when I hit the https://setfive.com/contact-us page with a POST though

image

**Review by adatta02 — APPROVED** [review]

**Pull request merged**